### ISSUE TITLE
Vysor android keyboard keeps crashing following desktop app update

### ISSUE THREAD

**Issue by Heretushi** [koush/vysor.io#429]
**Describe the bug**
Vysor android keyboard (2.1.8) keeps crashing when using Vysor Pro Version 2.2.2.

**To Reproduce**

1. Open Vysor Pro 2.2.2 on desktop.
2. Plug Android Phone.
3. Click view.
4. Vysor will display "Installing Vysor APK" (but it's already installed and up to date)
5. Phone screen will display.
6. Go to text field on phone, click in it. Vysor keyboard will NOT display.
7. Click on the Gboard "switch keyboard" button. Choose Vysor.
8. Gboard will start to disappear, but won't.
9. Repeat until Android nags you that "Vysor keeps crashing".

**Expected behavior**
I expect the notification "Vysor is hiding your keyboard", the keyboard to disappear and international keyboard to work properly, like last friday (2020/01/17).

**Screenshots**
Will provide on demand.

**Desktop (please complete the following information):**
 - Vysor Pro 2.2.2

**Android (please complete the following information):**
 - Device: Pixel 3
 - OS: Android 10

**Additional context**
Vysor Android App reports version 2.1.8. It's the one on the Google Play App. Uninstalling the android app before launching Vysor on the desktop will not correct the behavior. Every device, app and service has been rebooted prior to this bug report.

**Comment by Heretushi** [comment]
**Additional context (2)**
Using international keyboard or NOT does not change the behavior. The Android App keeps crashing.

**Comment by Heretushi** [comment]
**Additional context (2)**
When I keep repeating steps 6 to 9, Vysor will eventually stick around BUT international keyboard will not work properly. No accented characters will display even though the Vysor keyboard sticks around.

**Comment by koush** [comment]
Hey, thanks for this report. Looking into this today.

**Comment by koush** [comment]
If you know how, can you grab me a logcat of the crash as well?

**Comment by koush** [comment]
This is the crash I'm seeing:
```
01-20 13:03:33.694 16274 16274 D AndroidRuntime: Shutting down VM
01-20 13:03:33.694 16274 16274 E AndroidRuntime: FATAL EXCEPTION: main
01-20 13:03:33.694 16274 16274 E AndroidRuntime: Process: com.koushikdutta.vysor, PID: 16274
01-20 13:03:33.694 16274 16274 E AndroidRuntime: java.lang.IllegalStateException: IInputMethodPrivilegedOperations must be set at most once. privOps=com.android.internal.inputmethod.IInputMethodPrivilegedOperations$Stub$Proxy@38183d4
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at com.android.internal.inputmethod.InputMethodPrivilegedOperations$OpsHolder.set(InputMethodPrivilegedOperations.java:52)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at com.android.internal.inputmethod.InputMethodPrivilegedOperations.set(InputMethodPrivilegedOperations.java:94)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at android.inputmethodservice.InputMethodService$InputMethodImpl.initializeInternal(InputMethodService.java:481)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at android.inputmethodservice.IInputMethodWrapper.executeMessage(IInputMethodWrapper.java:168)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at com.android.internal.os.HandlerCaller$MyHandler.handleMessage(HandlerCaller.java:44)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:107)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:214)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7356)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
01-20 13:03:33.694 16274 16274 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
01-20 13:03:33.703  1350 16397 I DropBoxManagerService: add tag=data_app_crash isTagEnabled=true flags=0x2
```

**Comment by Heretushi** [comment]
Please just point me toward a guide to take a logcat and I will. :)

**Comment by Heretushi** [comment]
Found this : https://developer.android.com/studio/command-line/logcat

Will try tomorrow.

**Comment by Heretushi** [comment]
I had A LOT of text going on. I tried to isolate the interesting part, the most resembling yours. This is what I got :

```
01-20 22:06:51.335 25056 25056 E AndroidRuntime: FATAL EXCEPTION: main
01-20 22:06:51.335 25056 25056 E AndroidRuntime: Process: com.koushikdutta.vysor, PID: 25056
01-20 22:06:51.335 25056 25056 E AndroidRuntime: java.lang.IllegalStateException: IInputMethodPrivilegedOperations must be set at most once. privOps=com.android.internal.inputmethod.IInputMethodPrivilegedOperations$Stub$Proxy@4d3d28d
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at com.android.internal.inputmethod.InputMethodPrivilegedOperations$OpsHolder.set(InputMethodPrivilegedOperations.java:52)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at com.android.internal.inputmethod.InputMethodPrivilegedOperations.set(InputMethodPrivilegedOperations.java:94)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at android.inputmethodservice.InputMethodService$InputMethodImpl.initializeInternal(InputMethodService.java:481)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at android.inputmethodservice.IInputMethodWrapper.executeMessage(IInputMethodWrapper.java:168)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at com.android.internal.os.HandlerCaller$MyHandler.handleMessage(HandlerCaller.java:44)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:107)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:214)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7356)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
01-20 22:06:51.335 25056 25056 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by koush** [comment]
This should be fixed in v2.2.4. Can you verify? @Heretushi

**Comment by Heretushi** [comment]
I can verify that after a Vysor Pro auto-update (to 2.2.5) AND a Vysor reinstallation on the phone, this crash is fixed.
Strange behavior BEFORE the reinstallation of Vysor on the phone : even though Vysor was installed, it was absent from the keyboard choices when pressing the appropriate button.

I deem this issue resolved. Thanks koush. Welcome back to Vysor. :) 👍
